app/controllers/welcome_controller.rb

```diff
@@ -0,0 +1,10 @@
+class WelcomeController < ApplicationController
+
+  def index
+    @albums = Work.gen_top_ten_works('album')
```

Comment:
I think that this action would probably fit in the works controller. It's not wrong here, but it is an extra file to pay attention to.